lib/ruby_lsp/test_helper.rb

```diff
@@ -46,7 +46,7 @@ def with_server(source = nil, uri = Kernel.URI("file:///fake.rb"), stub_no_typec
         RubyLsp::Addon.addons.each(&:deactivate)
         RubyLsp::Addon.addons.clear
       end
-      T.must(server).run_shutdown
+      T.must(server).shutdown
```

Comment:
Aren't addons already deactivated in the condition above?

Reply:
Oh, I missed that we were invoking deactivation manually. In that case, we probably don't need this PR.